### PR TITLE
Add extra Perl requirements

### PR DESCRIPTION
**Pull request by hasegeli**

### CODE REVIEW

File: Bucardo/Installation.md [hunk 1]
```diff
@@ -40,8 +40,11 @@ sudo make install
 
 Other Perl prerequisites that may or may not be installed already on your system include:
 
+- `CGI`
```
Comment (jonjensen) [inline Bucardo/Installation.md]: This is only used by the bucardo-report, which is not a core part of Bucardo, and which I haven't used before. So it's not a prereq for Bucardo itself. Maybe just add ` (only needed for bucardo-report)`?

File: Bucardo/Installation.md [hunk 2]
```diff
@@ -40,8 +40,11 @@ sudo make install
...
 - `DBI`
 - `DBD::Pg`
+- `Encode::Locale`
+- `Pod::Parser`
```
Comment (jonjensen) [inline Bucardo/Installation.md]: These are only needed for developers building or testing Bucardo, not for running Bucardo itself. Again, some kind of annotation to that effect, or maybe put them in a different section?

**Review by jonjensen — COMMENTED** [review]
@hasegeli These aren't quite prerequisites. See inline notes.

**Comment by hasegeli** [comment]
I appended these at the end of the new Requirements page.

**Comment by jonjensen** [comment]
@hasegeli Thanks!

**Pull request merged**